Adyen/adyen-magento
# Update ApplePayController.php

**Description**
fix a Notice: Undefined property. this notice block payment on development environment.


**Tested scenarios**
`error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT`
- on iPhone connected to Safari with web inspector
- go to magento server with apple pay
- add product to card and go to checkout
- fill all required filed to get Payment method step.
- Select Apple Pay method
- Click to Continue button
- On review step click Apple Pay logo
- Should appear a popup with selecting credit card
- use TouchId or Passcode to send a payment
- See "Payment Not Completed"
- See in Network Tab of Safari request to  /adyen/applePay/sendPayment/ with php Notice in output.
